8-typography.rst

```diff
@@ -1542,6 +1542,48 @@ Citations
 				<cite>—<i epub:type="se:name.publication.book">The Rubaiyat of Omar Khayyam</i></cite>
 			</blockquote>
 
+Verses and Chapters of the Bible
+================================
+
+#.	Citations of passages from the Bible should include the name of the book, followed by the chapter number and the verse number.
```

Comment:
Re. phrasing, we shouldn't be saying `should be`. Instead, we say `are`, because we are describing how final ebooks look.

Can we merge the first child rule into this top level one? `... Followed by the chapter number, a colon, and the verse number`

Reply:
Note that the phrasing correction applies to this entire PR